downstream/assemblies/platform/assembly-ug-controller-jobs.adoc

```diff
@@ -0,0 +1,50 @@
+[id="controller-jobs"]
+
+= Jobs in {ControllerName}
+
+A job is an instance of {ControllerName} launching an Ansible playbook against an inventory of hosts.
+
+The *Jobs* list view displays a list of jobs and their statuses, shown as completed successfully, failed, or as an active (running) job. 
+The default view is collapsed (Compact) with the job name, status, job type, start, and finish times. 
+You can click the arrow (image:arrow.png[Arrow,15,15]) to expand and see more information. 
+You can sort this list by various criteria, and perform a search to filter the jobs of interest.
+
+image::ug-jobs-list-all-expanded.png[Jobs list expanded]
+
+From this screen you can complete the following tasks:
+
+* View details and standard output of a particular job 
+* Re-launch (image:rightrocket.png[Rightrocket,15,15]) jobs
+* Remove selected jobs
+
+The relaunch operation only applies to relaunches of playbook runs and does not apply to project or inventory updates, system jobs, and workflow jobs.
+When a job relaunches, you are directed to the *Jobs Output* screen as the job runs.
```

Comment:
Maybe: When a job relaunches, the * Jobs output * (page/screen/view) is displayed.